packages/core/src/component-playground/ComponentPlayground.stories.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { getPathForComponent } from '../../story-structure';
+
+import App from './ComponentPlayground';
+
+export default {
+  title: getPathForComponent('o', 'Examples'),
```

Comment:
Change layout to full screen 

`
  parameters: { layout: 'fullscreen' },
`

Reply:
Good catch -- will check that in. @matthewgallo This fixes the extra space we were seeing around the PageHeader, including the content showing through.

Reply:
I jumped in to help with some things, including adding this piece ^